client/src/components/parts/Header/index.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { Component, Fragment } from "react";
+import { BrowserRouter as Router, Link } from "react-router-dom";
+import PropTypes from "prop-types";
+import * as Logo from "./logo.png";
+import HamburgerButton from "../HamburgerButton";
+import "./style.css";
+
+function Menu({ show }) {
+  if (show)
+    return (
+      <Router>
+        <div className="header__menu">
+          <Link to="/profile" className="header__menu-text">
+            Profile
+          </Link>
+          <Link to="/new-post" className="header__menu-text">
+            Create Post
+          </Link>
+          <Link to="/posts/live" className="header__menu-text">
+            Live Posts
+          </Link>
+          <Link to="/posts/draft" className="header__menu-text">
+            Draft
+          </Link>
+          <a href="/logout" className="header__menu-text">
```

Comment:
use Link tag, not anchor tag
as a user, I don't want to wait the same time when open the site for logout.

Reply:
So the process of the user staying logged in and to log out will just to the frontend to worry about?

Reply:
i with @aminalakhsham  in this situation, you can handle whole process with ajax requests (fetch).